src/components/login/LoginForm.js

```diff
@@ -12,7 +12,7 @@ import style from '../../style';
 /** View component for displaying the login form to the user
  * @type {ILoginFormComponentProps}
  */
-export const LoginForm = ({
+export const LoginFormView = ({
```

Comment:
Unneeded export of view component, unless you plan on testing the view directly

Reply:
I currently has some basic tests dependent on this export